include/crypto/evp.h

```diff
@@ -396,13 +396,12 @@ static int cname##_ecb_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out, const uns
 #define BLOCK_CIPHER_func_cbc(cname, cprefix, kstruct, ksched) \
 static int cname##_cbc_cipher(EVP_CIPHER_CTX *ctx, unsigned char *out, const unsigned char *in, size_t inl) \
 {\
-        while(inl>=EVP_MAXCHUNK) \
-            {\
+        while(inl>=EVP_MAXCHUNK) {\
             cprefix##_cbc_encrypt(in, out, (long)EVP_MAXCHUNK, &EVP_C_DATA(kstruct,ctx)->ksched, ctx->iv, EVP_CIPHER_CTX_is_encrypting(ctx));\
             inl-=EVP_MAXCHUNK;\
             in +=EVP_MAXCHUNK;\
             out+=EVP_MAXCHUNK;\
-            }\
+        }\
```

Comment:
Having no space before the trailing `\` looks weird - is this fine?
Also some 10 lines below.

Reply:
I was just addressing a specific `checkpatch.pl` rule here. I'd rather stick to that in this PR.

Also, I think that having a space before the trailing `\` is not a rule in the OpenSSL coding style.

I'll fix the indention of course.